crates/exec-wasmtime/src/loader/compiled/mod.rs

```diff
@@ -20,15 +20,15 @@ impl Loader<Compiled> {
         let ctx = ctx.data_mut();
 
         // Set up environment variables.
-        for (k, v) in self.0.config.env.iter() {
```

Comment:
Is this change just intended to get rid of the explicit `.iter()`, or is the intent to change this from by-ref iteration to by-val? If it's the former, you can do `for (k, v) in &self.0.config.env {` and avoid taking references inside the loop.

Reply:
The intention was to move it.